src/Avatar.js

```diff
@@ -1,7 +1,7 @@
 import React, {useState, useCallback} from 'react';
-import {PlaceholderAvatar} from './PlaceholderAvatar';
+import {PlaceholderAvatar} from './components/PlaceholderAvatar/PlaceholderAvatar';
```

Comment:
if do `export default PlaceholderAvatar` you will import it like `import PlaceholderAvatar from '...'` it's a bit easier